ui/packages/catalog-ui-search/src/main/webapp/js/widgets/cesium.bbox.js

```diff
@@ -221,7 +221,7 @@ Draw.BboxView = Marionette.View.extend({
     if (
       rectangle &&
       !_.isUndefined(rectangle) &&
-      (rectangle.north !== rectangle.south && rectangle.east !== rectangle.west)
+      (rectangle.north > rectangle.south && rectangle.east !== rectangle.west)
```

Comment:
❓  out of scope for this ticket, but what happens when east and west are flipped? Not sure which one is supposed to be "greater", or if that logic applies at all for east and west

Reply:
So east and west _can_ be flipped because of the anti-meridian. But even with a bounding box _not_ over the anti-meridian, it doesn't seem to matter if east and west are flipped on both 2D and 3D maps 🤷‍♀

Reply:
sounds good to me!